### ISSUE TITLE
xgboost explain_weights fail

### ISSUE THREAD

**Issue by victordek** [TeamHG-Memex/eli5#272]
Found out a weird thing for xgboost:
```
from xgboost import XGBClassifier
from eli5 import explain_weights
import numpy as np

X = np.array([[1, 2]])
y = np.array([3])

model = XGBClassifier()
model.fit(X, y)

explain_weights(model)
```
Produced this:
```
boosting.py:29: DeprecationWarning: numpy.core.umath_tests is an internal NumPy module and should not be imported. It will be removed in a future NumPy release.
  from numpy.core.umath_tests import inner1d
Traceback (most recent call last):
  File "elixg.py", line 11, in <module>
    explain_weights(model)
  File "/home/someone/elissue/.py36/lib/python3.6/site-packages/singledispatch.py", line 210, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
  File "/home/someone/elissue/.py36/lib/python3.6/site-packages/eli5/xgboost.py", line 70, in explain_weights_xgboost
    booster, is_regression = _check_booster_args(xgb)
  File "/home/someone/elissue/.py36/lib/python3.6/site-packages/eli5/xgboost.py", line 225, in _check_booster_args
    booster = xgb.booster()
TypeError: 'str' object is not callable
```

eli5 works fine for lightgbm and other sklearn estimators.

Environment:
Python 3.6.5
packages obtained by `pip install eli5 xgboost`:
```
attrs==18.1.0
eli5==0.8
graphviz==0.8.4
Jinja2==2.10
MarkupSafe==1.0
numpy==1.15.0
pkg-resources==0.0.0
scikit-learn==0.19.2
scipy==1.1.0
singledispatch==3.4.0.3
six==1.11.0
tabulate==0.8.2
typing==3.6.4
xgboost==0.72.1
```

**Comment by noleto** [comment]
It was fixed but there was no new release since the commit: https://github.com/TeamHG-Memex/eli5/commit/ddf825d39ec9e3201b7f190cf79f23e76e37186e
You can build from sources to get the fix or make a workaround:

`explain_weights(model.get_booster())`
